test.js

```diff
@@ -131,4 +133,31 @@ describe('anymatch', function() {
       matchers.pop();
     });
   });
+
+  describe('windows paths', function() {
+    anymatch.__set__('platform', 'win32');
+
```

Comment:
you can pass `{unixify: true}` on the micromatch options to normalize paths. take a look at how I'm [doing something similar in micromatch](https://github.com/jonschlinkert/micromatch/blob/master/test/pattern-string.js#L27-L48)